src/error.rs

```diff
@@ -149,7 +140,7 @@ impl From<core::array::TryFromSliceError> for Unspecified {
 pub struct KeyRejected(&'static str);
 
 impl KeyRejected {
-    /// The value returned from <Self as std::error::Error>::description()
+    /// The value is used in Display
     pub fn description_(&self) -> &'static str {
```

Comment:
Again, I think `description_()` should be removed; its code should be inlined into the `Display` implementation.